driver/razermouse_driver.c

```diff
@@ -1281,6 +1281,115 @@ static ssize_t razer_attr_read_mouse_dpi(struct device *dev, struct device_attri
     return sprintf(buf, "%u:%u\n", dpi_x, dpi_y);
 }
 
+/**
+ * Write device file "dpi_stages"
+ *
+ * Sets the mouse DPI stages to the array of unsigned shorts written to this
+ * file.
+ * The number of DPI stages is hard limited by RAZER_MOUSE_MAX_DPI_STAGES.
+ *
+ * Each DPI stage is described by 4 bytes:
+ *   - 2 bytes (unsigned short) for x-axis DPI
+ *   - 2 bytes (unsigned short) for y-axis DPI
+ *
+ * buf is expected to contain n*4 bytes where n is the number of DPI stages.
+ * If count is not a multiple of 4 then the residual bytes will be ignored.
+ */
```

Comment:
From reading through this comment I don't get what should be written to this file, a short example would be good like in the read function below

Reply:
Done in 75e48541c07e15bc02af0475447338bf077dc8ae, let me know if this helps.